### PR TITLE
Set the common ActionMailer default_url_options only once

### PR DESCRIPTION
**Pull request by paroga**
There is no need to set options other than the foodcoop during every
change of the scope. This also makes sure that the correct options will
be loaded during testing to fix problems with newer rails versions.

### CODE REVIEW

File: spec/app_config.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@ default: &defaults
   tax_default: 6.0
   email_sender: noreply@minimal.test
 
+  host: localhost
+
```
Comment (wvengen) [inline spec/app_config.yml]: Why is this necessary?

Comment (paroga) [reply spec/app_config.yml]: newer rails versions will throw an error if the host is not set
